examples/frames/test/status_test.clj

```diff
@@ -2,15 +2,14 @@
   (:require
     [clojure.test :refer :all]
     [components]
-    [framework.config.core :as config]
     [framework.handler.core :refer [handler-fn]]))
 
 (deftest status-test
-  (let [deps (components/deps (config/env))
-        request {:uri            "/status"
-                 :request-method :get}
-        handle (handler-fn deps)
-        visit (-> (handle request)
-                  (update :body slurp))]
-    (is (= 200 (:status visit)))
-    (is (= "{\"status\":\"OK\"}" (:body visit)))))
+  (with-open [deps (components/->system components/app-cfg)]
+    (let [request {:uri            "/status"
+                   :request-method :get}
+          handle (handler-fn deps)
```

Comment:
could be good to call everywhere system instead of deps

Reply:
Its should not have the whole system (for example this test doesn't need the webserver) but only the dependencies. So it's wrong in many other ways :D